index.html

```diff
@@ -4,16 +4,78 @@
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>Danielle Baxter's Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
+
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <header>
+      <nav>
+        <a href="https://danibaxx.github.io/portfolio-website/">Home</a>
```

Comment:
Perhaps the home can link to the Resume and the Portfolio anchor tag link to your Portfolio?